modules/matterhorn-authorization-manager/src/main/java/org/opencastproject/authorization/xacml/manager/impl/AclServiceImpl.java

```diff
@@ -243,8 +243,12 @@ public boolean applyAclToSeries(String seriesId, AccessControlList acl, boolean
 
         for (MediaPackage mp : mediaPackages) {
           // remove episode xacml and update in archive service
-          if (assetManager != null)
-            assetManager.takeSnapshot(authorizationService.removeAcl(mp, AclScope.Episode));
+          try {
+            if (assetManager != null)
+              assetManager.takeSnapshot(authorizationService.removeAcl(mp, AclScope.Episode));
```

Comment:
Hi @slampunk 

Not related to your change but... have you got an idea why the Episode ACL is removed here? (instead of the Series ACL)

Reply:
@staubesv I have no idea tbh. I had assumed it was a design choice to keep mediapackage ACLs in lockstep with that of their container series'.

Reply:
I would assume the same. I guess the idea is that if someone edits a series' acl all episodes belonging to that series should get that acl.